README.md

```diff
@@ -30,7 +30,7 @@ The `cics-bundle-maven-plugin` currently supports the following CICS bundleparts
  * Maven is installed into your environment, or
  * You use a Java IDE that supports Maven, e.g. Eclipse, IntelliJ, VS Code...
  
- The CICS bundle deployment API is supported by the CMCI JVM server that must be set up in a WUI region. To use this make sure that:
+ The CICS bundle deployment API is supported by the CMCI JVM server that must be set up in a WUI region (consult the [CICS TS doc](https://www.ibm.com/support/knowledgecenter/en/SSGMCP_5.6.0/fundamentals/cpsm/cics-bundle-api.html) for details). To use this make sure that:
```

Comment:
I wonder whether we should use the Configuring task instead, as we're talking about the configuration requirements?
https://www.ibm.com/support/knowledgecenter/en/SSGMCP_5.6.0/configuring/cmci/config-bundle-api.html

Reply:
OK. I guess they can get back to how it works easily enough from the link at the top of that page.